src/internal/pjs/api_server.go

```diff
@@ -28,13 +28,6 @@ type apiServer struct {
 	pollInterval time.Duration
 }
 
-func newAPIServer(env Env) *apiServer {
-	return &apiServer{
-		env:          env,
-		pollInterval: 5 * time.Second,
```

Comment:
The pollInterval is defined but not initialized.

Reply:
Ahh good catch